frontend/components/core/ReButton.vue

```diff
@@ -221,6 +221,36 @@ export default {
   }
 }
 
+.button-quaternary {
+  @extend %button;
+  background: $lighter-color;
+  color: palette(grey, dark);
+  border: 1px solid palette(grey, smooth);
+  box-shadow: inset 0 -2px 6px 0 rgba(223,223,223,0.50);
+  .svg-icon {
+    margin-right: 1em;
+    vertical-align: middle;
+    fill: palette(grey, dark);
+  }
+  &:hover,
+  &:focus,
+  &:active,
+  &.active {
+    background-color: $lighter-color;
+    border: 1px solid darken(palette(grey, smooth), 10%);
+  }
+  &[disabled] {
+    background-color: $lighter-color;
+  }
+  &--small {
+    @extend .button-quaternary;
+    text-transform: none;
+    min-height: 30px;
+    line-height: 30px;
+    min-width: auto;
+  }
+}
+
```

Comment:
Why the core button style is changed here?

Reply:
it's a new button "button-quaternary"